dev/AppNotifications/ShellLocalization.h

```diff
@@ -9,8 +9,7 @@
 
 namespace Microsoft::Windows::AppNotifications::ShellLocalization
 {
-    const PCWSTR c_localMicrosoftFolder{ LR"(Microsoft)" };
-    const PCWSTR c_localWindowsAppSDKFolder{ LR"(WindowsAppSDK)" };
+    const PCWSTR c_localWindowsAppSDKFolder{ LR"(\Microsoft\WindowsAppSDK\)" };
```

Comment:
Do we have a unit test to verify that an image file is successfully created in the appropriate path? Strongly recommend that we add one after calling Register API